wdl-engine/src/stdlib/find.rs

```diff
@@ -30,8 +30,8 @@ fn find(context: CallContext<'_>) -> Result<Value, Diagnostic> {
         .coerce_argument(1, PrimitiveType::String)
         .unwrap_string();
 
-    let regex =
-        Regex::new(pattern.as_str()).map_err(|e| invalid_regex(&e, context.arguments[1].span))?;
+    let regex = Regex::new(pattern.as_str())
+        .map_err(|e| function_call_failed("find", &e, context.arguments[1].span))?;
```

Comment:
Could we replace all the `function_call_failed(<hardcoded str>,...` with something like `Self::function_name()`?

Reply:
It's a free function, so there's no `Self` context. At best perhaps a constant. I can do that.